scripts/ram_info.sh

```diff
@@ -16,16 +16,21 @@ get_percent()
 {
 	case $(uname -s) in
 		Linux)
-			# percent=$(free -m | awk 'NR==2{printf "%.1f%%\n", $3*100/$2}')
-			used_mem=$(free -g | awk '/^Mem/ {print $3}')
-			total_mem=$(free -h | awk '/^Mem/ {print $2}')
-			if (( $used_mem == 0 )); then
-				memory_usage=$(free -m | awk '/^Mem/ {print $3}')
-				echo $memory_usage\M\B/$total_mem\G\B
-			else
-				memory_usage=$(free -g | awk '/^Mem/ {print $3}')
-				echo $memory_usage\G\B/$total_mem\G\B
-			fi
+                        # percent=$(free -m | awk 'NR==2{printf "%.1f%%\n", $3*100/$2}')
```

Comment:
The indentation has changed

Reply:
Hi @ethancedwards8, I'm really not sure why it shows the changes like it does, because it does line up as expected in my PR. Anyways let med just look at it. =)

Reply:
Tracing the changes should be easier now. 👍

Reply:
Sure thing, looks good.